pkg/shell/base_index.js

```diff
@@ -33,6 +33,15 @@ function component_checksum(machine, component) {
         return "$" + machine.manifests[pkg][".checksum"];
 }
 
+function setDarkMode(documentElement, dark_mode) {
+    console.info("Set dark mode", dark_mode);
+    if (dark_mode) {
+        documentElement.classList.add('pf-theme-dark');
+    } else {
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17986-20221204-161200-7083ac82-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)